components/centraldashboard/app/api.ts

```diff
@@ -5,22 +5,22 @@ import {KubernetesService} from './k8s_service';
 
 /** API routes exposing information from Kubernetes */
 export function api(kubeConfig: KubeConfig): express.Router {
```

Comment:
It looks like we have different spacing styles for Typescript and it's not being enforced in the project by tslint. I'm using the Google Typescript standard which is a two-space indent.

 Do you want to revert this file since there are no semantic changes. I'll make sure we enforce the proper spacing on save in a future commit, otherwise we'll just keep flip-flopping back and forth each time one of us touches a file.

Reply:
Sure done!